src/classes/VOL_CTRL_PersonalSiteContactLookup_TEST.cls

```diff
@@ -104,6 +148,8 @@ private with sharing class VOL_CTRL_PersonalSiteContactLookup_TEST {
     }
 
     private static void TestPersonalSiteContactLookup() {
+        //disable captcha
+        UTIL_UnitTest.setupCaptchaSettings(false);
```

Comment:
If it is disabled by default, why do we need to disable it here?

Reply:
Safeguards the test against unexpected changes elsewhere by being explicit here.